packages/fx-core/src/plugins/solution/fx-solution/question.ts

```diff
@@ -88,6 +88,12 @@ export const AzureResourceApim: OptionItem = {
   description: "Azure Function App will be also selected to be published as an API",
 };
 
+export const AzureResourceKeyVault: OptionItem = {
+  id: "keyvault",
+  label: "Azure Key Vault",
+  description: "Description for Azure Key Vault",
```

Comment:
@MuyangAmigo  Please help provide description for Key Vault. It will be shown in:
![image](https://user-images.githubusercontent.com/1050213/143399946-3f1d8080-973e-48ba-8a6d-8710e721e869.png)

Reply:
@formulahendry How about this one: `Secure runtime application secrets with Azure Key Vault`

Reply:
Updated.